components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -64,6 +64,9 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) CALayer *shadowLayer;
 
+/** The shadow color of the @c shadowLayer. */
```

Comment:
Tip: Since this is `nonnull`, do you want to document the default?

Reply:
Done